From 7243001d5511922f293f0641cb8dbc0af4850dae Mon Sep 17 00:00:00 2001
From: Todd Lipcon <todd@cloudera.com>
Date: Fri, 18 Jun 2010 16:13:45 -0700
Subject: [PATCH 0316/1020] HDFS-1254. Enable append feature by default

Description: Changes dfs.support.append to "true" in hdfs-default.xml
Reason: Append/sync have been tested in CDH3b2 and are safe to use.
Author: Dhruba Borthakur
Ref: CDH-659
---
 src/hdfs/hdfs-default.xml |    7 ++-----
 1 files changed, 2 insertions(+), 5 deletions(-)

diff --git a/src/hdfs/hdfs-default.xml b/src/hdfs/hdfs-default.xml
index 1219e9e..b748b12 100644
--- a/src/hdfs/hdfs-default.xml
+++ b/src/hdfs/hdfs-default.xml
@@ -370,11 +370,8 @@ creations/deletions), or "all".</description>
 
 <property>
   <name>dfs.support.append</name>
-  <value>false</value>
-  <description>Does HDFS allow appends to files?
-               This is currently set to false because there are bugs in the
-               "append code" and is not supported in any prodction cluster.
-  </description>
+  <value>true</value>
+  <description>Does HDFS allow appends to files?</description>
 </property>
 
 <property>
-- 
1.7.0.4

